### DISCUSSION TITLE
How to provide negative_prompt in training

### DISCUSSION THREAD

**Discussion by imJunaidAfzal** [TheLastBen/fast-stable-diffusion#1564] [Q&A]
I'm looking for a way to provide negative_prompt while training for the faces, or other objects...

**Comment by TheLastBen** [comment]
The thing you can do is manually mask the parts of the image you don't want to include in the training, for example the background, you can simply replace the background with black color and it will be ignored during the training.

**Reply by remybonnav** [reply]
If you have large datasets that you don't want to process manually for background removal, You can even try to mask what you don't xant using
https://github.com/OPHoperHPO/image-background-remove-tool
